apps/vr-tests/src/stories/NewButton.stories.tsx

```diff
@@ -192,23 +192,23 @@ storiesOf('ActionButton', module)
   .addStory('Default', () => <ActionButton {...baseProps} />, { rtl: true })
   .addStory('Disabled', () => <ActionButton disabled {...baseProps} />);
 
-storiesOf('BaseButton', module)
-  .addDecorator(FabricDecorator)
-  .addDecorator(story => (
-    <Screener
-      steps={new Steps()
-        .snapshot('default', { cropTo: '.testWrapper' })
-        .hover('.ms-Button')
-        .snapshot('hovered', { cropTo: '.testWrapper' })
-        .mouseDown('.ms-Button')
-        .snapshot('pressed', { cropTo: '.testWrapper' })
-        .end()}
-    >
-      {story()}
-    </Screener>
-  ))
-  .addStory('Default', () => <BaseButton {...baseProps} />, { rtl: true })
-  .addStory('Disabled', () => <BaseButton disabled {...baseProps} />);
+// storiesOf('BaseButton', module)
+//   .addDecorator(FabricDecorator)
+//   .addDecorator(story => (
+//     <Screener
+//       steps={new Steps()
+//         .snapshot('default', { cropTo: '.testWrapper' })
+//         .hover('.ms-Button')
+//         .snapshot('hovered', { cropTo: '.testWrapper' })
+//         .mouseDown('.ms-Button')
+//         .snapshot('pressed', { cropTo: '.testWrapper' })
+//         .end()}
+//     >
+//       {story()}
+//     </Screener>
+//   ))
+//   .addStory('Default', () => <BaseButton {...baseProps} />, { rtl: true })
+//   .addStory('Disabled', () => <BaseButton disabled {...baseProps} />);
```

Comment:
Hmm what's the story here?

Reply:
I think I wrote that down in the PR description, but basically I'm removing `BaseButton` on this PR just to get it out for review of the other parts while we wait on getting #9436 in. Then I can merge that in here and get `BaseButton` again.

Reply:
I did read the description, I just didn't see the association with this test. ;)